### ISSUE TITLE
Multi-objective Property Specfications

### ISSUE THREAD

**Issue by ribsthakkar** [stormchecker/stormpy#31]
I am running Stormpy with an MDP, and it works without issue when my property string is a single objective. However, when I seem to use mutli-objective strings, the `stormpy.model_checking` function returns `None`.

Here is a snippet of my code for reference:
```
formula_str = "multi(Rmin=? [F \"goal\"], Pmax=? [F \"truegoal\"])"
formulas = stormpy.parse_properties(formula_str)
result = stormpy.model_checking(mdp, formulas[0], extract_scheduler=True) # returns None
```

 I am not sure if there is an error in my property string (which doesn't raise any errors when I use the parser module) or Stormpy doesn't support multiobjective checking yet?

**Comment by tquatmann** [comment]
Multi-objective model checking only produces a result for the initial state of the model (I'm assuming your model has a single initial state).
If you request a result for all states (which implicitly happens in your snippet), then the model checker can not handle the query and returns `None`.
To solve this, simply add `only_initial_states=True` to the `stormpy.model_checking(...)` call.

As a side note: 
Scheduler extraction is not yet supported for multi-objective model checking. In this particular case, the query results in a Pareto curve, i.e. a set of optimal points, each induced by a different scheduler. Hence, it is not even clear which of the possible optimal schedulers to extract.
